sunscreen/src/types/zkp/rns_polynomial.rs

```diff
@@ -254,4 +308,58 @@ mod tests {
 
         assert!(result.is_err());
     }
+
+    #[test]
+    fn can_scale_polynomial() {
+        #[zkp_program(backend = "bulletproofs")]
+        fn add_poly<F: BackendField>(
```

Comment:
I think this name was copy-pasted, as the function itself doesn't do any polynomial addition.

Reply:
Yep